### PR TITLE
Add ability to add or update an event in iOS

### PR DESCRIPTION
**Pull request by MaikuB**
Also fixes example code to cater for scenarios where events might have missing info like null description etc

### CODE REVIEW

File: device_calendar/example/lib/presentation/event_item.dart [hunk 1]
```diff
@@ -29,8 +29,8 @@ class EventItem extends StatelessWidget {
               child: new FlutterLogo(),
             ),
             new ListTile(
-                title: new Text(_calendarEvent.title),
-                subtitle: new Text(_calendarEvent.description)),
+                title: new Text(_calendarEvent.title ?? ''),
```
Comment (mkieres) [inline device_calendar/example/lib/presentation/event_item.dart]: Wouldn't this just work? As in, when null is provided then the text is basically empty?

File: device_calendar/lib/src/common/constants.dart [hunk 2]
```diff
@@ -2,5 +2,6 @@ class Constants {
   static const String fromJsonMapIsNull = 'The json object is null';
 
   static const String invalidArgument = 'Provided arguments are invalid';
-  static const String createOrUpdateEventArgumentRequirements = 'To create or update an event you must provide calendar id, event with title, start date and end date (where start date must be before end date)';
+  static const String createOrUpdateEventArgumentRequirements =
```
Comment (mkieres) [inline device_calendar/lib/src/common/constants.dart]: Is this a correct formatting?

Comment (MaikuB) [reply device_calendar/lib/src/common/constants.dart]: yes via flutter format

Comment (mkieres) [reply device_calendar/lib/src/common/constants.dart]: Sorry, I thought there's an extra line space between the variable name and its value.

You should be careful with flutter format though, as sometimes it breaks the logic of the app - situation that I had with chaining extension methods

**Pull request merged**